src/Application/Commands/ArchiveShoppingList/ArchiveShoppingListInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lindyhopchris\ShoppingList\Application\Commands\ArchiveShoppingList;
+
+interface ArchiveShoppingListInterface
```

Comment:
Required: for consistency, this should actually be called "ArchiveShoppingListCommandInterface", as the other command interfaces have the word "Command" in them. It's a really minor bit of feedback, but it's "Required" because as a team we'd want our code to be consistent - as it makes it more _predictable_, which is better for long term maintenance of the codebase.